### ISSUE TITLE
User reported ERROR nlp/slides/cfree-parsing.en.xhtml

### ISSUE THREAD

**Issue by abchugh** [slatex/sTeX-React#134]
A content ERROR was logged by "a user" at the following url:

https://courses.voll-ki.fau.de/course-notes/lbs?inDocPath=19xheo%7E51a76fad

## The issue as described by the user:
```
graph appearance has a problem!
```

## The text highlighted while reporting this issue:
```
I  
                                shoot  
                                the  
                                Wumpus  
                                Pronoun  
                                TransVerb  
                                Article  
                                Noun  
                                
                                
                                
                                
                                NP  
                                NP  
                                
                                
                                
                                VP  
                                
                                
                                S
```

### The selected text was in the following section hierarchy:

**_INNERMOST SECTION FIRST_**

1. GitLab: https://gl.mathhub.info/MiKoMH/AI/-/blob/main/source/nlp/slides/cfree-parsing.en.tex<br/>FetchURL: https://stexmmt.mathhub.info//:sTeX/fulldocument?archive=MiKoMH/AI&filepath=nlp/slides/cfree-parsing.en.xhtml&bindings=0_3_2jcafsxR0

2. GitLab: https://gl.mathhub.info/MiKoMH/LBS/-/blob/main/source/gf/sec/cfg-recap.en.tex<br/>FetchURL: https://stexmmt.mathhub.info//:sTeX/fulldocument?archive=MiKoMH/LBS&filepath=gf/sec/cfg-recap.en.xhtml&bindings=0_1_wxTr8

3. GitLab: https://gl.mathhub.info/MiKoMH/LBS/-/blob/main/source/gf/sec/intro.en.tex<br/>FetchURL: https://stexmmt.mathhub.info//:sTeX/fulldocument?archive=MiKoMH/LBS&filepath=gf/sec/intro.en.xhtml&bindings=0_1_227l4

4. GitLab: https://gl.mathhub.info/MiKoMH/LBS/-/blob/main/source/course/sec/symsyssem.en.tex<br/>FetchURL: https://stexmmt.mathhub.info//:sTeX/fulldocument?archive=MiKoMH/LBS&filepath=course/sec/symsyssem.en.xhtml&bindings=0_1_7SHW

5. GitLab: https://gl.mathhub.info/MiKoMH/LBS/-/blob/main/source/course/notes/notes.tex<br/>FetchURL: https://stexmmt.mathhub.info//:sTeX/fulldocument?archive=MiKoMH/LBS&filepath=course/notes/notes.xhtml

**Comment by kohlhase** [comment]
Indeed, but I think that this is an sTeX/RusTeX problem: This is how it should look: 
<img width="1180" alt="Screenshot 2023-11-01 at 20 33 25" src="https://github.com/slatex/sTeX-React/assets/4037538/988aee53-5db5-474a-aa44-3f08b8e5e597">
and here is how it does at first: 
<img width="582" alt="Screenshot 2023-11-01 at 20 36 22" src="https://github.com/slatex/sTeX-React/assets/4037538/8802da72-2396-4bbe-ae81-a53b5f0658d3">
and after you click somewhere: 
<img width="594" alt="Screenshot 2023-11-01 at 20 37 12" src="https://github.com/slatex/sTeX-React/assets/4037538/4dfc5109-f231-46e0-9d7f-18ba50c41afb">
I have no idea what this timing issue is about. Looks like a front-end thing.
